app/views/videos/show.html.haml

```diff
@@ -6,5 +6,30 @@
       .video_info.col-sm-3
         %header
           %h3= @video.title
-          %span Rating: (coming soon!)
+          %span Rating: (#{display_avg_video_stars(@video)})
         %p= @video.tagline
+
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_for [@video, @review] do |f|
+        %fieldset
+          .form-group
+            = f.label :rating, 'Rate this video'
+            .row
+              .col-sm-3
+                - possible_stars = (1..5).map { |n| [pluralize(n, 'Star'), n] }
```

Comment:
Logic like this is a good candidate for a helper method, it shouldn't be in the view.